src/tasks/taskinterfaces.ts

```diff
@@ -1,34 +1,29 @@
 import { IAction } from "../classes";
 
 // Base interfaces
-/** @hidden */

```

Comment:
why are we removing all these `@hidden` markings?

Reply:
Result of a bad merge. I couldn't remember if they were needed or not. I'll go back and put them in where they were removed.